sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_usage_v1/metadata.yaml

```diff
@@ -4,12 +4,9 @@ owners:
 - wichan@mozilla.com
 labels:
   incremental: true
-  schedule: daily
-scheduling:
-  dag_name: bqetl_monitoring
-  arguments: ["--date", "{{ ds }}"]
 bigquery:
   time_partitioning:
     type: day
     field: creation_date
     require_partition_filter: true
+deprecated: true
```

Comment:
Wouldn't it need to be addded to workgroup:deprecated?

Reply:
I'd love to see some docs around the deprecation process. `deprecated: true` is the right way to go about here since we have some mechanisms in place to also change the dataset permissions if there is a deprecated table part of the dataset

Reply:
I'm going to wait to merge this until we get the process established or cemented

Reply:
Just an FYI, this is [the proposal for the deprecation process](https://docs.google.com/document/d/1rnYmW21z8MM9saB0fHm5r9gIh8B8EYgTZSj-QIyMq5E/edit), the status is not approved yet though.

Reply:
And we will document the process, effects while we are doing this